src/components/ComingSoon.tsx

```diff
@@ -0,0 +1,42 @@
+import "./comingSoon.scss";
+import Noti from "../res/Noti.svg";
+import Bolt from "../res/Bolt.svg";
+import Back from "../res/Back.svg";
+import Features from "../res/Features.png";
+
+function ComingSoon() {
+  return (
+    <div className="section apps" id="apps">
+      <h1 className="coming-Header">Coming Soon</h1>
```

Comment:
The "apps" class and id should only be for the apps section, not the coming soon page